src/commands/core/asset/create.ts

```
@@ -2,226 +2,524 @@
+
+    // Upload metadata
+    const jsonSpinner = ora('Uploading metadata...').start();
+    const jsonUri = await uploadJson(umi, metadata);
+    jsonSpinner.succeed(`Metadata uploaded to ${jsonUri}`);
+
+    return { imageUri: imageUri as UploadFileRessult, jsonUri };
+  }
 
-      imageSpinner.succeed(`Image uploaded to ${imageUri.uri}`)
+  private async createAssetFromFiles(umi: Umi, options: CreateAssetOptions): Promise<AssetCreationResult> {
+    const jsonFile = JSON.parse(fs.readFileSync(options.json!, 'utf-8'));
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)